src/main/java/bot/parsers/InputParser.java

```diff
@@ -0,0 +1,69 @@
+package bot.parsers;
+
+public class InputParser {
+    private static final String SPLIT_AT_SPACE = " ";
+    private static final String SPLIT_AT_BY = " /by ";
+    private static final String SPLIT_AT_FROM = " /from ";
+    private static final String SPLIT_AT_TO = " /to ";
```

Comment:
Just a small consideration, but by including a space at by, from and to, would it cause an exception for inputs. such as "deadline /by22/08/13 ..." ? Could look at splitting with "/by" then trimming.